### PR TITLE
GEODE-4394: gfsh put command: change option --skip-if-exists to --if-…

### PR DESCRIPTION
**Pull request by sboorlagadda**
…not-exists

Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [x] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [x] Is your initial contribution a single, squashed commit?

- [x] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

File: geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/PutCommand.java [hunk 1]
```diff
@@ -53,7 +53,10 @@ public Result put(
       @CliOption(key = {CliStrings.PUT__VALUEKLASS},
           help = CliStrings.PUT__VALUEKLASS__HELP) String valueClass,
       @CliOption(key = {CliStrings.PUT__PUTIFABSENT}, help = CliStrings.PUT__PUTIFABSENT__HELP,
-          specifiedDefaultValue = "true", unspecifiedDefaultValue = "false") boolean putIfAbsent) {
+          specifiedDefaultValue = "true", unspecifiedDefaultValue = "false") boolean putIfAbsent,
```
[the remark this answers is not in the document]
Comment (sboorlagadda) [reply geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/PutCommand.java]: Changed it to be generic IFNOTEXISTS, but have to use a specific HELP string in-order to show that the synonym is deprecated.

File: geode-core/src/main/java/org/apache/geode/management/internal/cli/i18n/CliStrings.java [hunk 2]
```diff
@@ -80,11 +80,14 @@
   public static final String MEMBER = "member";
   public static final String MEMBERS = "members";
   public static final String IFEXISTS = "if-exists";
+  public static final String IFNOTEXISTS = "if-not-exists";
```
Comment (PurelyApplied) [inline geode-core/src/main/java/org/apache/geode/management/internal/cli/i18n/CliStrings.java]: I like the unification of redundant variables, but you haven't replaced the two other uses of `if-not-exists`.

File: geode-core/src/main/java/org/apache/geode/management/internal/cli/i18n/CliStrings.java [hunk 3]
```diff
@@ -1800,8 +1803,9 @@
   public static final String PUT__VALUE__HELP =
       "String or JSON text from which to create the value.  Examples include: \"manager\", \"100L\" and \"('value': 'widget')\".";
   public static final String PUT__PUTIFABSENT = "skip-if-exists";
-  public static final String PUT__PUTIFABSENT__HELP =
-      "Skip the put operation when an entry with the same key already exists. The default is to overwrite the entry (false).";
+  public static final String PUT__PUTIFNOTEXISTS = "if-not-exists";
```
Comment (PurelyApplied) [inline geode-core/src/main/java/org/apache/geode/management/internal/cli/i18n/CliStrings.java]: If you prefer the variable alias `PUT__*`, you could define this instead with 
```
  public static final String PUT__PUTIFNOTEXISTS = IFNOTEXISTS;`
```

The same applies to the `CREATE_REGION_*` one.

File: geode-core/src/main/java/org/apache/geode/management/internal/cli/i18n/CliStrings.java (hunk 2 above)
Comment (sboorlagadda) [reply geode-core/src/main/java/org/apache/geode/management/internal/cli/i18n/CliStrings.java]: @PurelyApplied Fixed it. Let me know if its good to go.

**Pull request merged**